app/views/admin/booths/_all_booths.csv.haml

```diff
@@ -0,0 +1,19 @@
+- headers = ['Booth ID',
+             'Title',
+             'Description',
+             'Reasoning',
+             'Submitter Name',
+             'Submitter Relationship',
+             'Website Url',
+             'State']
+= CSV.generate_line ['All booths']
+= CSV.generate_line headers
+- @booths.each do |booth|
+  = CSV.generate_line(	[booth.id,
+  						booth.title,
```

Comment:
Indentation here is not OK. Please put everything vertically aligned, and use spaces for your indentation.